### PR TITLE
update recipe

### PR DESCRIPTION
**Pull request by ocefpaf**
Testing the new format.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,32 +1,29 @@
...
+    - pip
     - cython
-    - numpy 1.8.*  # [not (win and (py35 or py36))]
-    - numpy 1.9.*  # [win and py35]
-    - numpy 1.11.*  # [win and py36]
-    - geos 3.6.2
-    - msinttypes  # [win and py<35]
+    - numpy
+    - geos
+    - msinttypes  # [win and py<14]
```
Comment (isuruf) [inline recipe/meta.yaml]: `win and vc<14`

Comment (ocefpaf) [reply recipe/meta.yaml]: Eagle eyes! Thanks!

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

File: recipe/bld.bat [hunk 2]
```diff
@@ -4,6 +4,10 @@ set INCLUDE=%LIBRARY_INC%;%INCLUDE%;%RECIPE_DIR%
 
 set GEOS_LIBRARY_PATH=%LIBRARY_BIN%\geos_c.dll
 
-%PYTHON% setup.py build_ext -I %LIBRARY_INC% -L %LIBRARY_LIB% -l geos_c
-%PYTHON% setup.py install --single-version-externally-managed --record record.txt
+"%PYTHON%" -m pip install --no-deps --ignore-installed . ^
+                          --global-option=build_ext ^
+                          --global-option="-I%LIBRARY_INC%" ^
+                          --global-option="-L%LIBRARY_LIB%" ^
+                          --global-option="-lgeos_c"
```
Comment (isuruf) [inline recipe/bld.bat]: Add `--verbose` here, otherwise the log is useless

**Pull request merged**